internal/database/online/cassandra/get.go

```diff
@@ -14,35 +14,47 @@ import (
 )
 
 func (db *DB) Get(ctx context.Context, opt online.GetOpt) (dbutil.RowMap, error) {
-	tableName := sqlutil.OnlineBatchTableName(opt.RevisionID)
+	var tableName string
+	if opt.Group.Category == types.CategoryBatch {
+		tableName = sqlutil.OnlineBatchTableName(*opt.RevisionID)
+	} else {
+		tableName = sqlutil.OnlineStreamTableName(opt.Group.ID)
+	}
 
 	query := fmt.Sprintf(`SELECT %s FROM %s WHERE %s = ?`,
 		strings.Join(opt.Features.Names(), ","),
 		tableName,
 		opt.Entity.Name,
 	)
 
-	rs := make(map[string]interface{})
-	if err := db.Query(query, opt.EntityKey).WithContext(ctx).MapScan(rs); err != nil {
+	datas := make(map[string]interface{})
```

Comment:
data is already plural, no need to add `s` at the end

Reply:
fixed!